### PR TITLE
fix(@snyk/fix): hide summary unless any failed, fixed or unresolved

### PR DESCRIPTION
**Pull request by lili2311**
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules

#### What does this PR do?
Only show summary if any items were fixed, failed, unresolved. If a plugin is called with empty test results return `No successful fixes` only.

Changes made as part of a design review of the CLI output

### CODE REVIEW

File: packages/snyk-fix/src/index.ts [hunk 1]
```diff
@@ -57,10 +58,14 @@ export async function fix(
   const meta = extractMeta(resultsByPlugin, exceptionsByScanType);
 
   spinner.start();
-  spinner.stopAndPersist({
-    text: 'Done',
-    symbol: meta.fixed === 0 ? chalk.red('✖') : chalk.green('✔'),
-  });
+  if (meta.fixed > 0) {
+    spinner.stopAndPersist({
+      text: 'Done',
```
Comment (lili2311) [inline packages/snyk-fix/src/index.ts:63]: only show happy  green Done if anything at all was fixed.

File: packages/snyk-fix/test/unit/__snapshots__/fix.spec.ts.snap [hunk 2]
```diff
@@ -58,7 +58,7 @@ Object {
     },
   },
   "fixSummary": "
-✖ No successful fixes
+ ✖ No successful fixes
```
Comment (lili2311) [inline packages/snyk-fix/test/unit/__snapshots__/fix.spec.ts.snap:61]: alignment

File: packages/snyk-fix/src/plugins/python/index.ts [hunk 3]
```diff
@@ -31,14 +33,18 @@ export async function pythonFix(
   );
   results.skipped.push(...notSupported);
 
-  spinner.succeed();
+  spinner.stopAndPersist({
+    text: spinnerMessage,
+    symbol: chalk.green('\n✔'),
```
Comment (lili2311) [inline packages/snyk-fix/src/plugins/python/index.ts:38]: added a new line

File: packages/snyk-fix/src/plugins/python/index.ts [hunk 4]
```diff
@@ -50,6 +56,10 @@ export async function pythonFix(
         results.failed.push(...failed);
         results.skipped.push(...skipped);
         results.succeeded.push(...succeeded);
+        spinner.stopAndPersist({
```
Comment (lili2311) [inline packages/snyk-fix/src/plugins/python/index.ts:59]: succeed after each project type

**Review by aarlaud — APPROVED** [review]

**Pull request merged**